runtime/lib/capi/RuntimeCAPI.cpp

```diff
@@ -137,17 +137,24 @@ void __quantum__rt__device(int8_t *spec, int8_t *value)
 
     const std::vector<std::string_view> args{reinterpret_cast<char *>(spec),
                                              reinterpret_cast<char *>(value)};
-    if (args[0] == "kwargs") {
+    if (args[0] == "rtd_kwargs") {
         Catalyst::Runtime::CTX->setDeviceKwArgs(args[1]);
         return;
     }
-    else if (args[0] == "backend") {
+    else if (args[0] == "rtd_name") {
+        Catalyst::Runtime::CTX->setDeviceName(args[1]);
+        return;
+    }
+    else if (args[0] == "rtd_lib") {
         RT_FAIL_IF(!Catalyst::Runtime::CTX->initDevice(args[1]),
                    "Failed initialization of the backend device");
+        if (Catalyst::Runtime::CTX->getDeviceRecorderStatus()) {
```

Comment:
@dime10 That's the update required after removing `status` from the `QuantumDevice` constructor.

Reply:
Nice, even better 💯